.drone.star

```diff
@@ -667,10 +665,10 @@ ocisSpecificTestSuites = [
 # minio mc environment variables
 minio_mc_environment = {
     "CACHE_BUCKET": {
-        "from_secret": "cache_public_s3_bucket",
+        "from_secret": "cache_s3_bucket",
```

Comment:
I think, `cache_public_s3_bucket`  can be left unchanged
```suggestion
        "from_secret": "cache_public_s3_bucket",
```

Reply:
Are you sure? Because in the ocis repo these have changed: https://github.com/owncloud/ocis/pull/6714/files

Reply:
Ahh, didnt notice that. But the secret is still available. @phil-davis is there any difference?

Reply:
It all has to match. If it is intended to upload files to the public cache (everyone will be able to publically read those files without auth from the cache server) `cache_public_s3_*` vars have to be used. If you want to use the private cache use `cache_s3_*`.

Reply:
The public cache is intended to be used for screenshots, logs etc. that needs to be added to PR comments. Again, files uploaded to the public cache will be readable for everyone, use it with caution.

Reply:
So, for e2e tracings `cache_public_s3_*` is best suited. So that we can view tracing without auth

Reply:
> I am still using `cache_public_s3_bucket` in https://github.com/owncloud/client/pull/11015/files

we are using s3_cache to upload gui test logs which should be viewable without any auth. so `cache_public_s3_bucket` seems right there too